src/contractTest/java/uk/gov/hmcts/reform/em/orchestrator/consumer/ConsumerTestUtil.java

```diff
@@ -61,6 +64,28 @@ public static void buildCcdCallbackResponse(LambdaDslObject body, Consumer<Lambd
             .array("warnings", arr -> {});
     }
 
+    public static DslPart createCloneRequestDsl() {
+        return newJsonBody(body -> buildCcdCallbackRequest(body, data ->
+            data.eachLike("caseBundles", bundle ->
+                bundle.object("value", value -> {
+                    buildCcdBundleDsl(value);
+                    value.stringMatcher("eligibleForCloning", "Yes|No", "yes");
+                })
+            )
+        )).build();
+    }
+
+    public static DslPart createCloneResponseDsl() {
+        return newJsonBody(body -> buildCcdCallbackResponse(body, data ->
+            data.minArrayLike("caseBundles", 2, bundle ->
+                bundle.object("value", value -> {
+                    buildCcdBundleDsl(value);
+                    value.stringMatcher("eligibleForCloning", "Yes|No", "no");
```

Comment:
consumer contract generation fails due to unmatched regex -> "Example "no" does not match regular expression "Yes|No""